plugins/dmxusb/src/dmxusbopenrx.cpp

```diff
@@ -0,0 +1,298 @@
+/*
+  Q Light Controller
```

Comment:
Q Light Controller Plus.
And I doubt Heikki has anything to do with a new file. Write my name instead cause sooner or later I'll edit this file.